style.css

```diff
@@ -0,0 +1,328 @@
+* {
+    box-sizing: border-box;
+    font-family: 'Lato', sans-serif;
+  }
+  body{
+      background-color: #ECF0F1;
+      color: #2C3E50;
+  }
+  .content{
+    max-width: 90%;
+    margin: 0 auto;
+    /* padding-left: 20px; */
+  }
+
+  body a{
+      color: #2C3E50;
+      text-decoration: none;
+  }
+  hr{
+      border: 1px solid #fc7e00;
+  }
+  h1{
+    color:white;
+    /* display: inline-block; */
+    /* vertical-align: top; */
+    float: left;
+    width: 240px;
+    padding: .5rem;
+  }
+  h2{
+      font-size: 2rem;
+  }
+  h3{
+    font-size: 1.5rem;
+  }
+  h4{
+    font-size: 1.75rem;
+  }
+  h5{
+    font-weight: bold;
+    font-size: 1.15rem;
+    line-height: 1.25rem;
+    margin: .75rem 0;
+  }
+  .header__image {
+    width: 60px;
+    margin: 1.25rem 0 0 1rem;
+    float: left;
+  }
+  header{
+      background-color: #112d49;
+  }
+  img{
+      max-width: 100%;
+  }
+
+  nav{
+    background-color: #2c3e50;
+    border-top: 3px #fc7e00ad solid;
+    border-bottom: 3px #fc7e00ad solid;
+      clear: both;
+      padding: .15rem;
+      text-align: center;
+  }
+    nav .menu{
+      padding: 0;
+  }
+nav li{
+    list-style: none;
+    display: inline-block;
+    color: #ecf0f1;
+    font-size: 1.25rem;
+    font-weight: bold;
+}
+
+nav a{
+    text-decoration: none;
+    color: #ecf0f1 ;
+    font-size: 1.25rem;
+    font-weight: bold;
+}
+.profile__info__mobile{
+    padding: 20px;
+    margin: 0;
+    display: block;
+}
+.profile__info__mobile li{
+list-style: none;
+text-align: left;
+font-size: 1.25rem;
+display: block;
+}
+
+.aboutMe{
+    padding: 0px 10px 0px 20px;
+}
+.aboutMe h3{
+    font-size: 2rem;
+}
+header .social a{
```

Comment:
This is likely too specific. Does the header matter or can you get rid of it in the selector?